packages/core/angular/src/wallet/wallet.store.ts

```diff
@@ -304,7 +353,9 @@ export class WalletStore extends ComponentStore<WalletState> {
                     return throwError(error);
                 }
 
-                return from(defer(() => adapter.sendTransaction(transaction, connection, options)));
+                return from(defer(() => adapter.sendTransaction(transaction, connection, options))).pipe(
+                    map((txId) => txId as TransactionSignature)
```

Comment:
Is this cast still needed? I see the other ones are gone.

Reply:
Good catch, it's not needed anymore.